LIMS_IMAGE/web/templates/orders/home_page.html

```diff
@@ -23,7 +39,7 @@ <h1>Client Home</h1>
     <p>You are not logged in</p>
     <a href="{% url 'login' %}">Log In</a>
     or
-    <a href="{% url 'accounts:registration' %}">Sign Up</a>
+    <a href="{% url 'accounts:registration' %}">Sign Up</a>    
```

Comment:
while you are at it, please remove the indeed extra space at the end of this line